### PR TITLE
146 orderbook scroll

### PR DESCRIPTION
**Pull request by lapinlapin**
#### What's this PR do?
Add scroll to buy/sell tables 
#### Where should the reviewer start?
views/OrderBook/OrderBookTable.vue
#### How should this be manually tested?
http://localhost:8080/#/
#### What are the relevant tickets?
#124 
#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/146-orderbook-scroll

### CODE REVIEW

File: src/views/Dashboard.vue [hunk 1]
```diff
@@ -50,7 +50,9 @@
         <Card
           class="lg:w-1/2"
           title="order book">
-          <div slot="body">
+          <div
+            slot="body"
+            :style="{'height': '100%'}">
```
Comment (roma219) [inline src/views/Dashboard.vue]: Simply use OrderBook directly in slot:
`<OrderBook slot="body"/>`

**Review by roma219 — CHANGES_REQUESTED** [review]
👍 
Just a few things:
1) see comment
2) Let's make a small padding at the bottom so there is no shadow visible when scrolled down fully.
image

File: src/views/Dashboard.vue (hunk 1 above)
Comment (lapinlapin) [reply src/views/Dashboard.vue]: fixed

**Comment by roma219** [comment]
@lapinlapin Sorry, it seems i wasn't clear enough... I didn't mean to remove shadows totally, in fact they were perfectly fine. I mean that there should be padding at the bottom (= height of the shadow) of the scrolling containers, so when scrolled fully down the shadow is not be visible. So that would kinda indicate that there is no more to scroll. 

And there are merge conflicts.

**Comment by roma219** [comment]
@lapinlapin please fix conflicts

**Pull request merged**